doc/ui/cmd_argument/detail_introduction.md

```diff
@@ -109,9 +109,9 @@
   - Load parameter from this pass to test.
   - type: int32 (default: -1).
 
-* `--test_period`
-  - Run testing every test_period train batches. If not set, run testing each pass.
-  - type: int32 (default: 1000).
+* `--test_period_while_training`
+  - Run test every so many train batches. If not 0, test log_period batches. If 0, test nothing.
+  - type: int32 (default: 0).
```

Comment:
Run test every test_period_while_training batches. If not 0, test log_period batches, else , test nothing.